qiita_db/util.py

```diff
@@ -656,6 +656,9 @@ def retrieve_filepaths(obj_fp_table, obj_id_column, obj_id):
         The name of the column that represents the object id
     obj_id : int
         The object id
+    sort : {'ascendent', 'descendent'}, optional
```

Comment:
This is for retrieving the children graph (descendents) or parents graph (ascendents), not sorting, so not sure that terminology fits here.